cmd/juno/juno.go

```diff
@@ -69,6 +73,8 @@ const (
 	p2pUsage                 = "enable p2p server"
 	p2PAddrUsage             = "specify p2p source address as multiaddr"
 	p2pBootPeersUsage        = "specify list of p2p boot peers splitted by a comma"
+	metricsUsage             = "enable prometheus endpoint"
+	metricsPortUsage         = "The port on which the prometheus server will listen for requests (default: 9090)"
```

Comment:
not sure if (default: 9090) is needed here as it is displayed twice, see here:
![image](https://github.com/NethermindEth/juno/assets/16260675/aa885e2d-42a2-44b5-bd33-7cf26cf87012)